mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -22,6 +22,14 @@ view: +subscriptions {
     primary_key: yes
   }
 
+  dimension: user_id {
+    hidden: yes
+  }
+
+  dimension: country {
+    hidden: yes
```

Comment:
Why hide the country?

Reply:
yeah, we have country and country_name, and we should only be exposing country_name in the explore.